core/messages/ja.json

```diff
@@ -171,7 +171,7 @@
             "messages": "メッセージ",
             "addresses": "住所",
             "newAddress": "New address",
-            "wishlists": "Wish lists",
+            "wishlists": "Wishlists",
```

Comment:
Not sure if we should edit this translation file

Reply:
According to our glossary, the correct version is `wishlist`: https://developer.bigcommerce.com/docs/start/design/ux-writing/glossary#x-y-z:~:text=word%2C%20no%20space.-,wishlist,-Do%20not%20use